handbook/ops/bizops/onboarding.md

```diff
@@ -0,0 +1,61 @@
+# BizOps onboarding
+
+Welcome! We're so excited to have you join the BizOps team. This document outlines the structure of your first few weeks at Sourcegraph.
+
+## Week 1
+
+### Get set up
+
+You'll have to get some basics set up in your first few days:
+
+- Complete the [general onboarding](../../people-ops/onboarding/general_onboarding.md). 
+
+### Understand company and team mission, values and goals
+
+- [Sourcegraph FY22 OKRs](https://docs.google.com/document/d/18d3sX38O6ephNuoHqZT9BwU1m1_FGuRfsTwYbc8lMV8/edit#heading=h.obnaanj0svtb)
+- [Sourcegraph values](../../../company/values.md)
+- [Sourcegraph master plan](../../../company/strategy.md)
+- [Sourcegraph direction (1 year plan)](../../../direction/index.md)
+- [Current BizOps projects](https://github.com/orgs/sourcegraph/projects/63)
+- [BizOps @ Sourcegraph deck](https://docs.google.com/presentation/d/1B8TIkOsFXUofLKcwu3ZKUuFgSe7si_HNFXj8Qne4_ug/edit#slide=id.gb969c13690_0_15)
+
+### Understand Sourcegraph communication and documentation standards
+
+- We're a handbook-first company. Learn all about the handbook on its [handbook page](../../../handbook/usage.md). 
+- Join relevant Slack channels. The main channels are [listed in the handbook](../../communication/team_chat.md#channels), but it's a good idea to look through all the channels in our Slack and join all that seem relevant! The channels that BizOps uses most heavily are #business-ops and #analytics, in addition to the channels of teams we work with. 
+- Learn about how we work. We're in all-remote company, which means [we work asynchronously as much as possible](../../../company/remote/index.md#all-remote) and lot of decision making happens through [RFCs](../../communication/rfcs/index.md).
+
+### Onboarding sessions
```

Comment:
Perhaps explain what this section means. Should they schedule these? How do they do so? Perhaps have links to the right people to connect with? Or just add a sentence at the top that says "your manager will help you schedule the following training sessions" or whatever.